bucketeer2/src/main/kotlin/io/bucketeer/sdk/android/internal/logger.kt

```diff
@@ -0,0 +1,90 @@
+package io.bucketeer.sdk.android.internal
+
+import android.util.Log
+import io.bucketeer.sdk.android.BKTLogger
+
+internal object LoggerHolder {
+  private val logHandlers: MutableList<BKTLogger> = mutableListOf()
+
+  fun addLogger(logger: BKTLogger) {
+    logHandlers.add(logger)
+  }
+
+  fun log(
```

Comment:
previus v1 implementation has a `isLogForUser` flag, but it seems like it has never been set `false`.

https://github.com/ca-dp/bucketeer-android-sdk/blob/1fbf26e07e979a19529ab448178143e8a46692dc/bucketeer/src/main/java/jp/bucketeer/sdk/log/SdkLogger.kt#L16